index.html

```diff
@@ -135,6 +136,22 @@ <h2>Johnson Family</h2>
   </div>
 
 
+  <div id="footer">
+    <div class="container">
+        
+            <div class="row">
+                <div class="col-md-4"></div>
+                <div class="col-md-4"><p id="imp"><center><p class="footertext">Made with <img src="images/heart.png"> by Girls in Open Source.</p></center></p></div>
+                <div class="col-md-4 text-right" id="right">
+                  <a href="https://github.com/fossiiita"><img src="images/github.png" class="img-circle" alt="the-brains"></a>
+                  <a href="https://www.facebook.com/opencodeiiita/"><img src="images/facebook.png" class="img-circle" alt="..."></a>
+                  <a href="mailto:iit2015113@iiita.ac.in"><img src="images/mail.png" class="img-circle" alt="..."></a>
```

Comment:
change the mail in mail to here to geekhaven one. change the github link to geekhaven org one.

Reply:
@Anupam-dagar done